packages/mdc-button/_mixins.scss

```diff
@@ -26,10 +26,10 @@
 
   @if ($light-or-dark == "light") {
     @include mdc-button-ink-color(text-primary-on-dark);
-    @include mdc-button-ripple((base-color: white, opacity: .32));
+    @include mdc-button-ripple((base-color: text-primary-on-dark, opacity: $mdc-filled-button-ripple-opacity));
```

Comment:
Can you do

@include mdc-button-ripple((base-color: map-get($mdc-theme-property-values, $text-primary-on-dark), opacity: $mdc-filled-button-ripple-opacity))

And then remove the @if statement from mdc-button-ripple

Reply:
We could do this. Though the argument I have here is since `theme` already agnostically taking either hex value or theme string, it makes it feel inconsistent from the standpoint of API.

On the other end, the other argument we had in the morning is it might worth we address it later in the ripple work and instead of `base-color` use `theme-color` temporarily since it is an existing API.

Reply:
I think using`theme-style` in the existing ripple config is better.

mdc-button-ripple should just take whatever "ripple config" we would use in mdc-ripple-* mixins

Reply:
The only thing preventing us from doing that right now is we're relying directly on the presence of `base-color` in this mixin, and `base-color` and `theme-style` are generally mutually exclusive, but yes, we'd like to ideally stop needing to do that. See my [other comment](https://github.com/material-components/material-components-web/pull/1280#discussion_r138396949)